src/Platform/Microsoft.Testing.Platform/IPC/Models/FileArtifactInfo.cs

```diff
@@ -0,0 +1,6 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Testing.Platform.IPC.Models;
+
+internal sealed record FileArtifactInfo(string FullPath, string DisplayName, string Description, string TestUid, string TestDisplayName, string SessionUid, string ModulePath) : IRequest;
```

Comment:
I would expect all requests to end with `Request` but I don't know if the current code is respecting that.